ggml/src/ggml-cuda/mmq.cu

```diff
@@ -137,6 +137,10 @@ bool ggml_cuda_should_use_mmq(enum ggml_type type, int cc, int64_t ne11) {
         return true;
     }
 
+#if defined(GGML_USE_MUSA)
+    return true;
+#endif // defined(GGML_USE_MUSA)
```

Comment:
It looks like `GGML_CUDA_FORCE_MMQ` is no longer working on `MUSA`.

Reply:
Not sure about this on x86, but it did work well from my tests on ARM64.

Reply:
This had been removed to fix running on x86